src/app/components/album-details/album-details.spec.js

```diff
@@ -0,0 +1,109 @@
+import angular from 'angular';
+import uiRouter from 'angular-ui-router';
+import chai from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import chaiAsPromised from 'chai-as-promised';
+
+chai.use(sinonChai);
+chai.use(chaiAsPromised);
+
+describe('Component: Album Details', function (done) {
+  beforeEach(angular.mock.module('albumDetails'));
+  beforeEach(angular.mock.module(uiRouter));
+ 
+  var element;
+  var scope;
+  beforeEach(angular.mock.inject(function($rootScope, $compile){
+    scope = $rootScope.$new();
+
+    scope.albums = '4MGDnutWZLdTCP3nmT1tgu';
+    element = angular.element('<album-details album="albums"></album-details>');
```

Comment:
In this cases, you can use `$componentController` to instanciate the controller and test it directly without having to render the component